Posit Infrastructure/New Posit Environment Quick Start Guide.md

```diff
@@ -0,0 +1,167 @@
+- You will only have access to the "bigpool" if you are a member of the `positwb_super_user` UNIX group.  You can request access to this group by emailing the [Data Science Team](phs.datascience@phs.scot).  You will also need Line Manager authorisation.
+- The Kubernetes container image has been upgraded to use **Ubuntu 22.04.5 LTS (Jammy Jellyfish)**, providing better support for Posit Workbench, R, Python, and associated packages.
+- A reduced-size version of TeX has been installed in the container image to decrease its size and reduce session startup times.
+
+---
+
+### Performance and Reliability Improvements
+- The root cause of **Status code 502/504 errors** has been addressed.  These errors should now only occur in exceptional circumstances.
+- RStudio projects now open reliably on the first attempt, even when saved on Stats.
+- Previously opened RStudio projects are now listed on the Posit Workbench homepage and can be opened directly from there without having to open an RStudio Pro session first.
+- Previous workarounds for installing geospatial R packages such as `{sf}` and `{leaflet}` are no longer required.  You can install and load these packages in the same way as any other R package.
+- R Packages that require C++14 or C++17 support, such as `{ranger}`, now install without error.
+
+---
+
+## 3. URLs for New Posit Applications
+- **Posit Workbench**: https://pwb-prod.publichealthscotland.org/
+- **Posit Package Manager**: https://ppm-prod.publichealthscotland.org/
+- **Posit Connect**: _Coming soon..._
+
+## 4. Logging in and Starting a Session
+
+- Follow the link above to the new production Posit Workbench environment and log in using your LDAP username and password.  These are the same credentials you have used for the old Posit Workbench environment.
+- The "New Session" dialog window has changed slightly.  On first requesting a new session, you are given a choice of the four available IDEs:
+
+![New Session Dialog 1](https://github.com/user-attachments/assets/000c2943-bd4b-4818-974a-679dbb8987a9)
+
+- As before, you need to enter the number of CPUs and Memory you need for your session.  **Important Change: you must specify the amount of memory you require in gigabytes (GB)**:
+
+![New Session Dialog 2](https://github.com/user-attachments/assets/bd593345-4b9a-49a4-a4c4-f52008524438)
+
+
+**Remember to select the minimum number of CPUs and amount of Memory required for the work you intend to do.**  In most cases, you only require more than 1 CPU if you are knowingly going to be running code using multiple threads or parallel processing.  The amount of Memory your require is dependent on the number of R packages you intend to use and the size of the dataset you will be working with.  There is further guidance available on the PHS Data Science Knowledge Base at https://public-health-scotland.github.io/knowledge-base/docs/Posit%20Infrastructure?doc=Memory%20Usage%20in%20SMR01.md
```

Comment:
```suggestion
**Remember to select the minimum number of CPUs and amount of Memory required for the work you intend to do.**  In most cases, you only require more than 1 CPU if you are knowingly going to be running code using multiple threads or parallel processing. The amount of memory you require is dependent on the number of R packages you intend to use and the size of the dataset you will be working with.  There is further guidance available on the PHS Data Science Knowledge Base at https://public-health-scotland.github.io/knowledge-base/docs/Posit%20Infrastructure?doc=Memory%20Usage%20in%20SMR01.md
```

Also is this really correct?
> The amount of memory you require is dependent on **the number of R packages you intend to use**

Reply:
```suggestion
**Remember to select the minimum number of CPUs and amount of memory required for the work you intend to do.**  In most cases, you only require more than 1 CPU if you are knowingly going to be running code using multiple threads or parallel processing.  The amount of memory you require is primarily dependent on the size of the dataset you will be working with.  There is further guidance available on the PHS Data Science Knowledge Base at https://public-health-scotland.github.io/knowledge-base/docs/Posit%20Infrastructure?doc=Memory%20Usage%20in%20SMR01.md
```

The number of R packages you intend to use will have an impact, but the primary impact on memory is the size of dataset, so I've updated the text to reflect this.  Does it read ok @Moohan ?